src/starship/README.md

```diff
@@ -0,0 +1,38 @@
+# Starship
```

Comment:
This will be overwritten by GH Action, how do I clean it up myself?

Reply:
Well... there is JS code, but it's in external repo and seems does not have human friendly CLI interface?